test/e2e/operator_test.go

```diff
@@ -313,6 +313,88 @@ func TestMutation(t *testing.T) {
 
 }
 
+func TestMutationPickMin(t *testing.T) {
```

Comment:
A small nit. Can you merge TestMutationPickMin with TestMutation under TestMutation and create a list of tests with name, pod and activeDeadlineSeconds as the params?

Reply:
I also merge TestNoMutation with the other two because all of them are same and can be boiled into one table driven test function.